main/lp/learnpathItem.class.php

```diff
@@ -3627,8 +3627,18 @@ public function scorm_update_time($total_sec = 0)
             error_log("total_time: $total_time");
         }
 
+        $lp_table = Database::get_course_table(TABLE_LP_MAIN);
+        $lp_id = intval($this->lp_id);
+        $sql = "SELECT * FROM $lp_table
+                    WHERE id = '$lp_id' AND c_id = $course_id";
```

Comment:
Remove the single quotes around $lp_id -> it is an integer, so don't assign it as if it was a string ;-)